rpc_test.go

```diff
@@ -0,0 +1,15 @@
+package rpc
+
+import (
+	"encoding/json"
+	"fmt"
+	"testing"
+)
+
+func TestRPC(t *testing.T) {
+	rpc := NewRPC("https://corgi-rpc.codechain.io/")
+	rpc.Ping(nil, nil)
+
+	res, _ := json.Marshal(rpc.Version(nil, nil))
```

Comment:
What is the role of `json.Marshal` here?

Reply:
I just tested that version() returns right format. I'll remove this because this is not necessary.